helpers/requests/requests.go

```diff
@@ -0,0 +1,87 @@
+package requests
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+)
+
+// ConstructGetURL Constructs a URL to use in a GET request.
+// Queries are passed as a map of string arrays
+func ConstructGetURL(
+	rawURL string, queries map[string][]string, format string, values ...interface{}) (*url.URL, error) {
+
+	urlPath, err := url.Parse(rawURL)
+	if err != nil {
+		fmt.Printf("Unable to parse url %q", rawURL)
+		return nil, err
+	}
+
+	urlPath.Path = fmt.Sprintf(format, values...)
+	var q url.Values = queries
+	urlPath.RawQuery = q.Encode()
+
+	return urlPath, nil
+}
+
+// GetAndParseJSON Makes a request to urlPath with basic auth using user and token.
+//	Attempts to parse the JSON response, returning it in "result" if successful.
```

Comment:
nit: Not returning anything. You're altering the true value

Also, I've found that referring to parameter names in the comments is not that sustainable as barely any refactoring tool will update the names there. Suggest to instead refer to their meaning/usage instead. Here we could even skip the `urlPath` as it's IMO obvious what it's used for.

Also2, basic auth is always user+password.

```suggestion
// GetAndParseJSON makes a HTTP request with basic auth.
// On success the response body will be unmarshalled as JSON.
```